WordPress/openverse-frontend
# Use v-show instead of v-if for width-based condition

## Fixes


Fixes #882 by @sarayourfriend

## Description


This PR does two things:
- replace `v-show` directives with a condition based on width(`isSmall`) with `v-if` to ensure that the server-rendered markup matches the client-rendered markup. I'm not sure that this is the best solution for all of these places, but it fixes the warnings.
- Replace the helpText options api based computed with a composition api based computed. I'm not sure it's needed, but I added it because the warning specifically was about `t` not being defined (I guess it's `i18n` `$t`?)

## Testing Instructions

Open the audio search page by going to http://localhost:8443/search/audio?q=bird on a desktop and look at the console. It should not display any warnings. 

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
